packages/components/src/internal/components/forms/input/UserSelectInput.tsx

```diff
@@ -18,23 +18,34 @@ interface UserSelectInputProps extends Omit<SelectInputProps, 'delimiter' | 'loa
 export const UserSelectInput: FC<UserSelectInputProps> = props => {
     const { notifyList, permissions, useEmail, ...selectInputProps } = props;
 
-    const loadOptions = useCallback(async () => {
-        let options_;
-
-        try {
-            const users = await getUsersWithPermissions(permissions);
-            options_ = users
-                .map(v => ({
-                    label: v.displayName,
-                    value: notifyList ? v.displayName : useEmail ? v.email : v.userId,
-                }))
-                .toArray();
-        } catch (error) {
-            console.error(error);
-        }
-
-        return options_;
-    }, [notifyList, permissions, useEmail]);
+    const loadOptions = useCallback(
+        async (input: string) => {
+            let options_;
+            const input_ = input?.trim().toLowerCase();
```

Comment:
Don't we using use an underscore at the beginning for these internal avoid-conflict names instead of at the end?  I don't care deeply, of course; just seems weird as a suffix.

Reply:
In my experience (though mostly Python) a prefixed underscore means the variable is meant to be private/internal to the class or module you're defining it in, a suffix underscore is used to eliminate reserved word / existing word issues.

Reply:
Revised variable names.